lib/compat/wordpress-6.1/block-patterns/footer-with-large-font-size.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Footer: Footer with Large Font Size.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'      => _x( 'Footer with Large Font Size', 'Block pattern title' ),
+	'blockTypes' => array( 'core/template-part/footer' ),
+	'categories' => array( 'footer' ),
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"15px","padding":{"top":"30px","right":"30px","bottom":"30px","left":"30px"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"left"}} -->
+					<div class="wp-block-group alignfull" style="padding-top:30px;padding-right:30px;padding-bottom:30px;padding-left:30px;"><!-- wp:site-title {"textAlign":"center","fontSize":"large"} /-->
+					
+					<!-- wp:paragraph {"align":"center"} -->
+					<p class="has-text-align-center">Proudly Powered by <a href="https://wordpress.org">WordPress</a></p>
```

Comment:
Is there a reason why the `data-type="URL"` and `data-id` are used inconsistently, on only some external links?
Does the pattern benefit from these?

Reply:
I'll defer to @beafialho. If there are no specific reasons, we should just be consistent and boring. I am not familiar with this `data-type` / `data-id` practice.

Reply:
I took the liberty of removing those attributes along with other i18n changes in 0836d91882caaccd3a7984be38362a8d0f074b8a